tests/debug/allocate_deallocate.cpp

```diff
@@ -10,9 +10,9 @@
 
 int main() {
   auto &rm = umpire::ResourceManager::getInstance();
-  auto alloc = rm.getAllocator("NO_OP");
+  auto alloc = rm.getAllocator("HOST");
```

Comment:
Why are we changing this? I thought No_Op was needed here?

Reply:
This was accidental.  However, I thought that the stuff under "debug" was changeable?  I didn't think they were actually tests that ran.

Reply:
This can be anything. Like Marty said, it's not tests that run, it's just little code snippets useful for debugging.

Reply:
Ah, ok. It's been so long since this was created that I forgot! Ha :p